### ISSUE TITLE
Learning Rate Visualization with AdaGrad Optimization

### ISSUE THREAD

**Issue by djebm2** [NVIDIA/DIGITS#645]
AFAIK, AdaGrad consists of Adapting the learning rate throughout the learning process. From what DIGITS shows me, it seems that the accuracy converges faster than SGD. This indicates that AdaGrad in DIGITS leverages the adaptive learning rate as it should be. However, The learning rate curve has the same evolution as SGD. 

**AdaGrad**

![adagrad](https://cloud.githubusercontent.com/assets/6205921/13884531/b03fecf2-ed2e-11e5-924e-13f16e63cf4d.png)

**SGD**

![sgd](https://cloud.githubusercontent.com/assets/6205921/13884536/b264149a-ed2e-11e5-9d92-16b5c9e1ef2b.png)

**Common Learning Rate evolution**

![learning_rate](https://cloud.githubusercontent.com/assets/6205921/13884538/b723540a-ed2e-11e5-986e-7fc02a7f95b6.png)

**Comment by gheinrich** [comment]
The learning rate is expected to be the same between `SGD` and `ADAGRAD`. The difference is in the weight update formula. Have a look at this [page](http://caffe.berkeleyvision.org/tutorial/solver.html) for more information.

**Comment by djebm2** [comment]
Ok. Thx.
